onadata/apps/logger/tests/data_migration/common.py

```diff
@@ -12,13 +14,22 @@ def remove_whitespaces(string):
     return string.replace('\n', '').replace(' ', '')
 
 
-class MigrationTestCase(TestCase):
-    def setUp(self):
-        self.user = self.create_user('johnny_cash')
-        self.xform = self.create_xform(fixtures.form_xml_case_1)
-        self.xform_new = self.create_xform(fixtures.form_xml_case_1_after)
-        self.survey = self.create_survey(self.xform_new)
-        self.setup_data_migrator(self.xform, self.xform_new)
+class CommonTestCase(TestCase):
```

Comment:
we'll have to refactor those tests, making new class for each new input is not the best idea

Reply:
ok, i made a ticket for that